shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -187,14 +182,56 @@ public Intent build(@NonNull Context context) {
 
   @Override
   protected void onCreate(@Nullable Bundle savedInstanceState) {
+    switchLaunchThemeForNormalTheme();
```

Comment:
What specific disconnect could happen here? If we were showing an activity in launch theme and no status bar etc, this onCreate happens at a point when we're potentially handing off the OS's launch drawable showing to our own 'splash view' showing right? If we switch the theme right here, would the status bar appear in the middle of an otherwise seamless transition? 

Should we document that perhaps people might want to create custom hooks to delay this theme switching until they call transitionToFlutter?

Reply:
I'm not sure about the specific visual artifacts that you're referring to. Can you provide some kind of example code or repro steps so I can see what you're concerned about?